docs/guides/python-flask-feature-flags.mdx

```diff
@@ -46,7 +50,7 @@ Python3 app.py
 
 If you navigate to http://127.0.0.1:5000 in your browser, you should see "Hello World!".
 
-Next, we'll add a new endpoint to our Flask app - one that we'll eventually use a feature flag to make accessible to only some users. Add this code below your last route ("Hello World"!) but above the app.run call:
+Next, we'll add a new endpoint to our Flask app—one that we'll eventually use a feature flag to make accessible to only some users. Add this code below your last route ("Hello World"!) but above the app.run call:
```

Comment:
ngl i like that spaces here

Reply:
I guess (for now) we can do whatever for docs. For instance, I left the titles in title case.

On WWW, and what I'm used to, instead of using a dash with spaces (which can be considered "incorrect" but is also "how devs type"), we use an em dash with no spaces as per the Chicago Manual of Style.

Sorry to nerd out, just wanted to explain myself.